core/src/test/java/org/elasticsearch/action/fieldcaps/FieldCapabilitiesTests.java

```diff
@@ -106,4 +109,75 @@ static FieldCapabilities randomFieldCaps() {
             randomAlphaOfLengthBetween(5, 20), randomBoolean(), randomBoolean(),
             indices, nonSearchableIndices, nonAggregatableIndices);
     }
+
+    @Override
+    protected MutateFunction<FieldCapabilities> getMutateFunction() {
```

Comment:
the one thing that comes to mind is: why does this need to be a function? I should have brought this up earlier, but it could just be a mutate method with one argument?